[bug] LinkedIn component not working

`com.bytechef.component.exception.ProviderException: 426: {"status":426,"code":"NONEXISTENT_VERSION","message":"Requested version 20250401 is not active"}`


**Stack Trace**
```
com.bytechef.platform.workflow.worker.exception.TriggerExecutionException: com.bytechef.component.exception.ProviderException: 426: {"status":426,"code":"NONEXISTENT_VERSION","message":"Requested version 20250401 is not active"}
at com.bytechef.platform.workflow.worker.trigger.handler.AbstractTriggerHandler.handle(AbstractTriggerHandler.java:72)
at com.bytechef.platform.workflow.worker.TriggerWorker.doExecuteTrigger(TriggerWorker.java:164)
at com.bytechef.platform.workflow.worker.TriggerWorker.lambda$onTriggerExecutionEvent$0(TriggerWorker.java:91)
at com.bytechef.platform.workflow.worker.executor.TriggerWorkerExecutor.submit(TriggerWorkerExecutor.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:112)
at com.bytechef.tenant.concurrent.TenantThreadPoolTaskExecutor.lambda$getTenantCallable$0(TenantThreadPoolTaskExecutor.java:70)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
at io.micrometer.context.ContextSnapshot.lambda$wrap$0(ContextSnapshot.java:91)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
at java.base/java.lang.VirtualThread.run(VirtualThread.java:456)
Caused by: ConfigurationException{entityClass=interface com.bytechef.component.definition.ActionDefinition, errorKey=109, errorMessageArguments=null, inputParameters=null} com.bytechef.exception.ConfigurationException: com.bytechef.component.exception.ProviderException: 426: {"status":426,"code":"NONEXISTENT_VERSION","message":"Requested version 20250401 is not active"}
at com.bytechef.platform.component.aspect.TokenRefreshAspect.wrapException(TokenRefreshAspect.java:246)
at com.bytechef.platform.component.aspect.TokenRefreshAspect.handleException(TokenRefreshAspect.java:107)
at com.bytechef.platform.component.aspect.TokenRefreshAspect.handleTokenRefresh(TokenRefreshAspect.java:79)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl$$SpringCGLIB$$0.executeTrigger(<generated>)
at com.bytechef.platform.component.facade.TriggerDefinitionFacadeImpl.executeTrigger(TriggerDefinitionFacadeImpl.java:129)
at com.bytechef.platform.workflow.worker.trigger.handler.AbstractTriggerHandler.handle(AbstractTriggerHandler.java:63)
... 15 more
Caused by: com.bytechef.component.exception.ProviderException: 426: {"status":426,"code":"NONEXISTENT_VERSION","message":"Requested version 20250401 is not active"}
at com.bytechef.component.exception.ProviderException.getProviderException(ProviderException.java:146)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.lambda$executeProcessErrorResponse$1(TriggerDefinitionServiceImpl.java:234)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.executeProcessErrorResponse(TriggerDefinitionServiceImpl.java:236)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl$$SpringCGLIB$$0.executeProcessErrorResponse(<generated>)
at com.bytechef.platform.component.context.HttpClientExecutor$1.intercept(HttpClientExecutor.java:458)
at com.github.mizosoft.methanol.Methanol$InterceptorChain.forward(Methanol.java:1153)
at com.github.mizosoft.methanol.Methanol.send(Methanol.java:401)
at com.bytechef.platform.component.context.HttpClientExecutor.execute(HttpClientExecutor.java:115)
at com.bytechef.platform.component.context.ContextImpl$HttpImpl$ExecutorImpl.execute(ContextImpl.java:336)
at com.bytechef.component.linkedin.trigger.LinkedInNewPostTrigger.poll(LinkedInNewPostTrigger.java:155)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.executePollingTrigger(TriggerDefinitionServiceImpl.java:449)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.lambda$doExecuteTrigger$1(TriggerDefinitionServiceImpl.java:513)
at java.base/java.util.Optional.map(Optional.java:260)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.doExecuteTrigger(TriggerDefinitionServiceImpl.java:513)
at com.bytechef.platform.component.service.TriggerDefinitionServiceImpl.executeTrigger(TriggerDefinitionServiceImpl.java:257)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:82)
at com.bytechef.platform.component.aspect.TokenRefreshAspect.handleTokenRefresh(TokenRefreshAspect.java:77)
... 27 more
```

--- a/server/libs/modules/components/linkedin/src/main/java/com/bytechef/component/linkedin/connection/LinkedInConnection.java
+++ b/server/libs/modules/components/linkedin/src/main/java/com/bytechef/component/linkedin/connection/LinkedInConnection.java
@@ -88,7 +88,7 @@ public class LinkedInConnection {
                 .apply((connectionParameters, context) -> ApplyResponse.ofHeaders(
                     Map.of(
                         "X-Restli-Protocol-Version", List.of("2.0.0"),
-                        "Linkedin-Version", List.of("202504"),
+                        "Linkedin-Version", List.of("202604"),
                         AUTHORIZATION, List.of(BEARER + " " + connectionParameters.getRequiredString(ACCESS_TOKEN))))));
 
     private LinkedInConnection() {

